pajackconnect

```diff
@@ -183,7 +185,7 @@ case "$1" in
 		reset
 		;;
 	*)
-		echo $"Usage:\n\t $0 {restart|start|stop|reset}\n"
+		printf "Usage:\n\t $0 {restart|start|stop|reset} [--no-auto-connect]\n"
```

Comment:
Changed to printf for a more consistent behaviour (https://stackoverflow.com/questions/8467424/echo-newline-in-bash-prints-literal-n)